game/scripts/npc/abilities/faceless_void_time_lock_oaa.txt

```diff
@@ -41,18 +41,18 @@
       "03"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "chance_pct"                                      "10 15 20 25 25 25"
+        "chance_pct"                                      "15 20 25 30 35 40"
```

Comment:
everything i said on the non_oaa version should apply here. Cooldown should be more than .2 (if u make it .4 stun duration i'd make the cooldown 1 second since you have to consider void can go octarine to reduce the cooldown of this spell.)

Reply:
The whole point of the void bash changes is to make him actually get good damage from time lock so he can carry. Any cooldown around 1 second will ruin this.